src/mcts/node.cc

```diff
@@ -293,7 +293,7 @@ void Node::SortEdges() {
 }
 
 void Node::MakeTerminal(GameResult result, float plies_left, Terminal type) {
-  SetBounds(result, result);
+  if (type != Terminal::TwoFold) { SetBounds(result, result); }
```

Comment:
clang-format indent style

Reply:
What does this mean, and what is the suggested change?

Reply:
Although the style guide is not absolutely clear about this, at least for consistency either don't add braces here or put `SetBounds(result, result);` on the next line.

Reply:
Removed the braces.